src/utils/http/cdfHttpClient.ts

```diff
@@ -74,10 +74,18 @@ export class CDFHttpClient extends RetryableHttpClient {
     delete filteredHeaders[name];
     return filteredHeaders;
   }
+
+  private disposableHeaders: HttpHeaders = {};
+
   constructor(baseUrl: string) {
     super(baseUrl, cdfRetryValidator);
   }
 
+  public addDisposableHeader(name: string, value: string) {
```

Comment:
I think `addOneTimeHeader` is a better name

Reply:
fixed